tripy/tests/test_examples.py

```diff
@@ -101,9 +129,29 @@ def test_examples(example, sandboxed_install_run):
 
             code = str(block)
             if block.has_marker("test: expected_stdout"):
-                print("Checking command output against expected output: ", end="")
                 out = statuses[-1].stdout.strip()
-                matched = re.match(dedent(code).strip(), out)
+                # expected = dedent(code).strip()
+                expected_outs = dedent(code).split("====")
```

Comment:
This change looks like it was based on an older branch

Reply:
Regex was breaking for nanogpt so I had to revert back to this change. I will fix it once this PR is merged.